### ISSUE TITLE
1.19 Regression with --stdin

### ISSUE THREAD

**Issue by dairyisscary** [prettier/prettier#6891]
**Environments:**
- Prettier Version: 1.19.0
- Running Prettier via: CLI
- Runtime: Node v10.14
- Operating System: MacOs

**Steps to reproduce:**
I did a clean install in a new directory of prettier:
```sh
$ yarn install prettier
$ cat test.js | prettier --stdin --stdin-filepath test.js
[error] regeneratorRuntime is not defined
```

Seems like the built "binaries" of `node_modules/prettier/bin-prettier.js` `node_modules/prettier/index.js` and `node_modules/prettier/third-party.js` all have this undefined name `regeneratorRuntime` presumably from a babel generator transformation, probably only used during stream/stdin reading. I double checked that 1.18.2 does not have this issue.

**Comment by dairyisscary** [comment]
after a bit of digging, it appears to be the transformed `async` function in [get-stream](https://github.com/sindresorhus/get-stream/blob/master/index.js#L12). 

i can't quite grok how babel configuration is specified for prettier's build process (somewhere in the `scripts/` directory I think?).

**Comment by fisker** [comment]
babel configuration will support corejs when #6866 get merged

**Comment by fisker** [comment]
working on this, I'll add a test for that

**Comment by lydell** [comment]
Fixed in 1.19.1.
